source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -349,10 +350,49 @@ def _check_response(cls, response, uri):
             raise TechnicalError('error in autocomplete request')
 
     @classmethod
-    def response_marshaler(cls, response_bragi, uri=None):
+    def _clean_response(cls, response, depth=3):
+
+        def _clear_object(obj):
+            if isinstance(obj, list):
+                del obj[:]
+            elif isinstance(obj, dict):
+                obj.clear()
+
+        def _manage_depth(_key, _value, _depth):
```

Comment:
This feels too generic, more than it is in kraken...
This code feels to clean :stuck_out_tongue_closed_eyes: 
Let's just compare results obtained in jormun using bragi vs. those using kraken (that's what you're supposed to do in tests)